policyengine_canada/parameters/gov/provinces/bc/tax/income/credits/disability/additional_amount/eligible_age.yaml

```diff
@@ -0,0 +1,12 @@
+description: British Columbia provides an additional disability credit to filers below this age
+values:
+  2023-01-01: 18
+metadata:
+  unit: year
+  period: year
+  label: British Columbia disability credit additional amount age eligibility
```

Comment:
label: British Columbia disability credit eligible age threshold

Reply:
It's providing additional disability credit for filers under this age, so I think 'British Columbia additional disability credit eligible age limit' describes it more accurately.